patches/@graphql-eslint__eslint-plugin@3.17.0.patch

```diff
@@ -0,0 +1,136 @@
+diff --git a/esm/estree-converter/utils.js b/esm/estree-converter/utils.js
```

Comment:
this patch will conflict with locally installed graphql-eslint that will lint hive monorepo schema/documents?

Reply:
I think it will work fine? but we need to double check or adjust the patch when we add graphql-eslint